.github/workflows/consecutive.yml

```diff
@@ -1,33 +1,74 @@
-name: "GNU Parallel: Python 3.8, Node.js 16, allure"
+name: "Run compatibility tests (python, node.js, openzeppelin)"
```

Comment:
just a note: looks like the multi-job workflow is not about running compatibility tests :)

Reply:
I think "Run <smth>" is more clearly than "GNU Parallel"